### DISCUSSION TITLE
How do I set something to always run Sandboxed? Also C:\Sandbox not found?

### DISCUSSION THREAD

**Discussion by clavzno** [sandboxie-plus/Sandboxie#2719] [Q&A Feedback] [answered]
On the app it shows the file/path to a box but when I go to that file/path (even with hidden items: show) there's nothing of the kind?

Also is there a setting that allows me to have a folder and app always sandboxed? Additionally, can I just locate the folder in File Explorer and import the folder there?

Then again given that I'm unable to find C:\Sandbox, what I mainly wanted to do can't be done. 

Any thoughts?

![image](https://user-images.githubusercontent.com/76804492/221406951-47cccf0a-b329-4802-80f1-2c5b358a0205.png)

**Comment by offhub** [comment, marked as answer]
> even with hidden items: show

That setting is only to show disabled sandboxes in the interface list.

> allows me to have a folder and app always sandboxed?

[ForceProcess](https://sandboxie-plus.github.io/sandboxie-docs/Content/ForceProcess.html)
[ForceFolder](https://sandboxie-plus.github.io/sandboxie-docs/Content/ForceFolder.html)

> Then again given that I'm unable to find C:\Sandbox, ...

Please open the [Sandboxie.ini](https://sandboxie-plus.github.io/sandboxie-docs/Content/SandboxieIni.html) file and look at the location set to the `FileRootPath` key under `[GlobalSettings]`.
Also, if no application has been sandboxed, the Sandbox folder may not have been created yet.

**Reply by clavzno** [reply]
Thank you!
